x-pack/plugins/index_management/__jest__/client_integration/home/indices_tab.test.ts

```diff
@@ -262,9 +276,14 @@ describe('<IndexManagementHome />', () => {
       await actions.clickManageContextMenuButton();
       await actions.clickContextMenuOption('openIndexMenuButton');
 
-      // A refresh call was added after closing an index so we need to check the second to last request.
```

Comment:
Should we keep part of this comment explaining that a refresh request will be made after opening an index?